.gitignore

```diff
@@ -123,6 +123,19 @@ fabric.properties
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+### JDK ###
+.idea
+out
+server
```

Comment:
Looking at gitignore patterns [here](https://www.atlassian.com/git/tutorials/saving-changes/gitignore), writing just `server` means that it matches both files named `server` and directories named `server`. Is this intended?

Reply:
yeah